common/djangoapps/student/models.py

```diff
@@ -1002,7 +1005,6 @@ def __unicode__(self):
         ).format(self.user, self.course_id, self.created, self.is_active)
 
     @classmethod
-    @transaction.atomic
```

Comment:
@tasawernawaz you can find out by checking the edxapp [database configuration](https://github.com/edx/configuration/blob/master/playbooks/roles/edxapp/defaults/main.yml#L687-L694) in the configuration repo. As you can see, we do use ATOMIC_REQUESTS, so requests made to the [EnrollmentListView](https://github.com/edx/edx-platform/blob/master/common/djangoapps/enrollment/views.py#L284) will be processed within transactions. Before changing this behavior for the view, you need to verify that data isn't created by the view in a way that could leave the database in a dirty state if a query fails.